source/ways-of-working/environments/palo-alto-external-ip-addresses.md.erb

```diff
@@ -0,0 +1,38 @@
+---
+title: Palo Alto External IP addresses
+last_reviewed_on: 2022-10-19
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+Please note:- IP address allow lists are not a recommended approach, see further in this article by GDS with guidance on why we try not to do this anymore: [A whitelisting approach for Cloud APIs](https://technology.blog.gov.uk/2017/01/03/a-whitelisting-approach-for-cloud-apis/)
```

Comment:
could you try use the warning text from https://github.com/hmcts/hmcts.github.io/blob/09ef9126c89837f04f6adcfa12d1b4006ea74651/source/ways-of-working/new-component/secrets-management.html.md.erb#L15

you'll need to run locally to make sure syntax is right and the link works

in the current setup I think 'Please note:- IP address' will look funny.

Can you also attach a screenshot please?